app/models/person.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Person < ApplicationRecord
+  has_paper_trail
+
```

Comment:
Should we consider adding this to the `ApplicationRecord` so models have versioning by default?

Reply:
do we want to audit every model including locations etc? Otherwise I agree it would make sense to have a common base class for auditable things